src/track/beatgrid.cpp

```diff
@@ -29,6 +30,24 @@ class BeatGridIterator : public BeatIterator {
         return beat;
     }
 
+    virtual bool isBar() const {
+        // TODO(JVC) - Stub
+        return false;
+    }
+
+    virtual bool isPhrase() const {
```

Comment:
isPhraseBeat

Reply:
BeatGrid is going to disappear so I will apply the change on the refurbished Beats class.